cypress/jenkins/cypress.config.jenkins.ts

```diff
@@ -109,9 +112,23 @@ export default defineConfig({
     setupNodeEvents(on, config) {
       require('cypress-mochawesome-reporter/plugin')(on);
       require('@cypress/grep/src/plugin')(config);
+
+      // Load Accessibility plugin if configured
+      if (process.env.TEST_A11Y) {
+        require('../../cypress/support/plugins/accessibility').default(on, config);
```

Comment:
I'd suggest to adopt error handling here in a try catch. if any of these modules fail to load (e.g., due to undefined env vars), it could crash Cypress without useful logging.

something like
```
try {
  require('../../cypress/support/plugins/accessibility').default(on, config);
} catch (err) {
  console.warn('Accessibility plugin failed to load:', err);
}
```